### PR TITLE
accept arrays as well.

### PR DESCRIPTION
**Pull request by nmn**

### CODE REVIEW

**Comment by jeffmo** [comment]
Thanks @nmn ! Looks like some of the libdef tests are failing with this change.
Can you fix those up before we merge?

**Comment by nmn** [comment]
Can we get a CONTRIBUTING.md file in there? The testing setup is kind of confusing.

**Comment by jeffmo** [comment]
> Can we get a CONTRIBUTING.md file

Great idea. I will write one a little later today.

**Comment by jeffmo** [comment]
In the meantime, you can run the tests locally with `cd cli && npm install && npm run build && node dist/cli.js run-tests classnames`. I'll include a simpler script to do this with the `CONTRIBUTING.md` doc.

File: definitions/classnames_v2.x.x/flow_>=v0.23.x/classnames_v2.x.x.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
+
+type Classes = string | [class: string]: bool } | Array<string>
```
Comment (jeffmo) [inline definitions/classnames_v2.x.x/flow_>=v0.23.x/classnames_v2.x.x.js]: Can we namespace this so it doesn't conflict as a global? Generally I've been recommending namespacing like `$npm$classnames$Classes`

Comment (nmn) [inline definitions/classnames_v2.x.x/flow_>=v0.23.x/classnames_v2.x.x.js]: Oh. I thought there were no global a if I didn't use `declare`

Comment (jeffmo) [inline definitions/classnames_v2.x.x/flow_>=v0.23.x/classnames_v2.x.x.js]: Ah -- So in declaration files, anything outside a `declare module {}` is considered global (this is how you can define actually-expected globals to Flow)

Comment (nmn) [inline definitions/classnames_v2.x.x/flow_>=v0.23.x/classnames_v2.x.x.js]: Again I thought I had to use
`declare var x = ...
declare type TypeName = ...`

In any case, I'll namespace this.

**Comment by jeffmo** [comment]
thanks!

**Pull request merged**